netmanager/src/AppRoutes.js

```diff
@@ -62,119 +63,39 @@ const AppRoutes = () => {
             component={AnalyticsDashboard}
             layout={MainLayout}
           />
-          <PrivateRoute
-            exact
-            path="/admin/users"
-            component={UserList}
-            layout={MainLayout}
-          />
-          <PrivateRoute
-            component={CandidateList}
-            exact
-            layout={MainLayout}
-            path="/candidates"
-          />
-          <PrivateRoute
-            component={Settings}
-            exact
-            layout={MainLayout}
-            path="/settings"
-          />
+          <PrivateRoute exact path="/admin/users" component={UserList} layout={MainLayout} />
+          <PrivateRoute component={CandidateList} exact layout={MainLayout} path="/candidates" />
+          <PrivateRoute component={Settings} exact layout={MainLayout} path="/settings" />
 
-          <PrivateRoute
-            path="/device/:deviceName"
-            component={DeviceView}
-            layout={MainLayout}
-          />
-          <PrivateRoute
-            exact
-            path="/locate"
-            component={Map}
-            layout={MainLayout}
-          />
-          <PrivateRoute
-            exact
-            path="/map"
-            component={OverlayMap}
-            layout={MainLayout}
-          />
-          <PrivateRoute
-            component={Account}
-            exact
-            layout={MainLayout}
-            path="/account"
-          />
-          <PrivateRoute
-            exact
-            path="/manager/map"
-            component={ManagerMap}
-            layout={MainLayout}
-          />
-          <PrivateRoute
-            exact
-            path="/manager/stats"
-            component={ManagerStats}
-            layout={MainLayout}
-          />
-          <PrivateRoute
-            exact
-            path="/sites"
-            component={SiteRegistry}
-            layout={MainLayout}
-          />
-          <PrivateRoute
-            exact
-            path="/sites/:id"
-            component={SiteView}
-            layout={MainLayout}
-          />
-          <PrivateRoute
-            exact
-            path="/airqlouds"
-            component={AirQloudRegistry}
-            layout={MainLayout}
-          />
-          <PrivateRoute
-            exact
-            path="/airqlouds/:id"
-            component={AirQloudView}
-            layout={MainLayout}
-          />
+          <PrivateRoute path="/device/:deviceName" component={DeviceView} layout={MainLayout} />
+          <PrivateRoute exact path="/locate" component={Map} layout={MainLayout} />
+          <Route exact path="/map">
+            <MainLayout>
+              <OverlayMap />
+            </MainLayout>
+          </Route>
```

Comment:
@FaithDaka I have set the map route to be a public route

Reply:
So I guess we are good to go here? @Codebmk 
I can not sense any pending requests on this PR...

cc: @FaithDaka / @123MwanjeMike

Reply:
Yeap it is